pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaAvroMessageDecoder.java

```diff
@@ -101,7 +99,8 @@ public GenericRow decode(byte[] payload, int offset, int length, GenericRow dest
         md5ToAvroSchemaMap.put(md5String, schema);
       } catch (Exception e) {
         schema = defaultAvroSchema;
-        LOGGER.error("error fetching schema from md5 String", e);
+        LOGGER.error("error fetching schema from md5 String from url {}", schemaRegistryBaseUrl, e);
```

Comment:
It'd be good to have the MD5 as well as the full URL that is used to fetch the schema. Can this be changed to output something like `Error fetching schema from URL http://.../id=1234`?

Also, since we throw an exception, `schema = defaultAvroSchema` does nothing, as `schema` is a local.